kms/main.tf

```diff
@@ -0,0 +1,15 @@
+resource "aws_kms_key" "encryption" {
+  description         = "KMS key for encryption within ${var.environment} environment"
+  enable_key_rotation = true
+  tags = merge(
+  var.common_tags,
```

Comment:
Have you run terraform fmt?

I'm going to add a card to the backlog about adding a Jenkins job to check this.

Reply:
thanks, done